eva-accession-release-automation/setup.py

```diff
@@ -7,6 +7,9 @@ def get_requires():
     with open(os.path.join(os.path.dirname(__file__), "run_release_in_embassy", "requirements.txt"), "rt") as req_file:
         for line in req_file:
             requires.append(line.rstrip())
+    with open(os.path.join(os.path.dirname(__file__), "gather_clustering_counts", "requirements.txt"), "rt") as req_file:
+        for line in req_file:
+            requires.append(line.rstrip())
```

Comment:
I agree. Is there a way to inherit a common top-level requirements.txt file and only add module specific dependencies in each module similar to Java POMs?

Reply:
Very true.  It should be possible to have requirements files that refer directly to other requirements files, but I'm having trouble getting it to work...

Anyway I think the path of least resistance here is have a single top-level requirements file, since the dependencies are almost the same and it corresponds with how we actually use them (shared virtualenv for everything in this directory).